### PR TITLE
Update ur.js

### PR DESCRIPTION
**Pull request by jdurand**
Urdu is RTL

### CODE REVIEW

File: addon/config/ur.js [hunk 1]
```diff
@@ -1,2 +1,5 @@
 import en from './en';
-export default en;
+
+export default {
+  rtl: true
+};
```
Comment (jamesarosen) [inline addon/config/ur.js]: You'll need some pluralization rules. If you want the ones from English (1 = "one", everything else is "other"), you could do
```js
import en from './en'
export default {
  pluralForm: en.pluralForm,
  rtl: true
}
```

**Comment by jamesarosen** [comment]
Change looks good. Something is wrong with the build.

**Comment by jdurand** [comment]
I think Travis just has a little cold. He'll get there.

**Comment by jdurand** [comment]
Looks like it's a proper flu after all...

**Comment by jamesarosen** [comment]
jamesarosen/ember-i18n has been deprecated in favor of [ember-intl](https://github.com/ember-intl).

**Pull request closed without merging**